about.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,400;0,700;0,900;1,900&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+        integrity="sha384-Bfad6CLCknfcloXFOyFnlgtENryhrpZCe29RTifKEixXQZ38WheV+i/6YWSzkz3V" crossorigin="anonymous" />
+    <link rel="stylesheet" href="style.css">
+    <title>About page 2022 United Nations Climate Change Summit</title>
+</head>
+
+<body>
+    <ul class="top">
+        <li><i class="fab fa-facebook-f"></i></li>
+        <li><i class="fab fa-twitter"></i></li>
+        <li class="top-logo">English</li>
+        <li class="top-logo">My page</li>
+        <li class="top-logo">Log out</li>
+    </ul>
+    <header class="logo">
+        <div class="logo-img"><a href="index.html"><img src="img/logo.png" width="100px" alt="summit-logo"></a></div>
+        <ul class="nav-list">
+            <li class="portfolio-link-nav">
+                <a class="portfolio-link-a-nav active" href="about.html">About</a>
```

Comment:
- This should link to the `index` file as already pointed out. Please make this change